database.js

```diff
@@ -27,7 +27,14 @@ const base = {
         if (err) throw err;
         callback(result);
       });
-  }
+  },
+  removeEntryFromCollection: function(collection, filter, callback) {
+    const writeResult = db.collection(collection)
+      .remove({ userTo }, true);
```

Comment:
This is removing information. A solution would be add a new taco rather than remove one but with a new field on the object that dictates if it's a negative or a positive taco. Ideally this would just be a new collection, so we don't keep adding fields for new taco behaviours (let's say we have a "mega" taco in the future that counts as 5) and that collection would store the occurrence of that new emoji.